rn-get-polyfills.js

```diff
@@ -13,4 +13,5 @@ module.exports = () => [
   require.resolve('./Libraries/polyfills/console.js'),
   require.resolve('./Libraries/polyfills/error-guard.js'),
   require.resolve('./Libraries/polyfills/Object.es7.js'),
+  require.resolve('./Libraries/polyfills/performance.js'),
```

Comment:
Do we need to have the polyfill? I think we really want to remove polyfills instead of adding more... what if we just added a JS module that people could require normally if they want to use it?

Reply:
I looked through the code, and I noticed that if Systrace is enabled, `Libraries/Performance/Systrace.js` would override `global.performance`. What if we load performance module in `InitializeCore.js`, and combine `performance.now` with all the methods from Systrace into one module that would define `performance`?

Reply:
Why can't we access `nativePerformanceNow`? It should be installed already, right?

Forgive the ignorant question, but - what would happen if, instead of installing `global.nativePerformanceNow`, we just actually installed `global.performance.now` directly instead? Would that be a disaster?

Reply:
I have no idea why I initially thought we cannot access `nativePerformanceNow` there. I moved the polyfill to Core instead.

The rest of `performance` implementation is in JS (it's part of `Systrace`), so I think it would be better to keep `now` in JS as well. Also, it's nice to have a fallback in case the executor has no `performance.now` implementation.